### DISCUSSION TITLE
Example for i18n-routing + i18next

### DISCUSSION THREAD

**Discussion by stfnsr** [vercel/next.js#18704] [Ideas]
I'd like to contribute an example for i18n-routing w/ i18next.
Currently there exists no example for i18n-routing with translations, and neither for i18next.
There is a link to next-i18next (https://github.com/vercel/next.js/tree/canary/examples/with-next-i18next), but it is comparatively complex, and the extra layer is not neccessary anymore, thanks to the new routing.

Would that be of interest?

**Comment by kachkaev** [comment]
There is a planned update to `next-i18next`, which should allow to simplify the example. See  https://github.com/isaachinman/next-i18next/issues/869

**Comment by stfnsr** [comment]
Thx, this actually sounds like a good way forward!
Will look into next-i18next then :)

**Comment by fillon** [comment]
Any progress?
